pkg/operator/certrotation/label.go

```diff
@@ -0,0 +1,61 @@
+package certrotation
+
+import (
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const (
+	// ManagedCertificateTypeLabelName marks config map or secret as object that contains managed certificates.
+	// This groups all objects that store certs and allow easy query to get them all.
+	// The value of this label should be set to "true".
+	ManagedCertificateTypeLabelName = "auth.openshift.io/managed-certificate-type"
+)
+
+type CertificateType string
+
+var (
+	CertificateTypeCABundle CertificateType = "ca-bundle"
+	CertificateTypeSigner   CertificateType = "signer"
+	CertificateTypeTarget   CertificateType = "target"
+	CertificateTypeUnknown  CertificateType = "unknown"
+)
+
+// LabelAsManagedConfigMap add label indicating the given config map contains certificates
+// that are managed.
+func LabelAsManagedConfigMap(config *v1.ConfigMap, certificateType CertificateType) {
```

Comment:
AddLabel.... 

Mutating funcs should have a verb in Go.

Reply:
Label is not a verb ?